nerfstudio/data/datamanagers/full_images_datamanager.py

```diff
@@ -61,6 +61,8 @@ class FullImageDatamanagerConfig(DataManagerConfig):
     """Specifies the image indices to use during eval; if None, uses all."""
     cache_images: Literal["cpu", "gpu"] = "cpu"
     """Whether to cache images in memory. If "cpu", caches on cpu. If "gpu", caches on device."""
+    cache_images_type: Literal["uint8", "float32"] = "uint8"
```

Comment:
maybe keep the old `float32` so no surprises / keep existing behavior?  that said, nerfstudio currently does not even support bigger bit depth https://github.com/nerfstudio-project/nerfstudio/blob/55b7a2c8ef3d2cc8a7eb7e66c67ec0278057df72/nerfstudio/data/datasets/base_dataset.py#L72

Reply:
You made a good point. I can update the PR to reflect the original behavior.